src/lib/robot/Robot.ts

```diff
@@ -14,7 +14,7 @@ export class Robot {
   f: Facing;
 
   static place(x: number, y: number, f: Facing) {
-    if (!Object.values(Facing).includes(f)) {
+    if (!Object.values(Facing).includes(f) || f === 4) {
```

Comment:
Invalid facings are all facings less than Facing._LENGTH, right?

Reply:
Anything less than Facing._LENGTH is valid: 0, 1, 2, 3, and then 4 + are invalid.
This could probably look way better, but it was the quickest way to fix that test 😁